cluster-autoscaler/provisioningrequest/orchestrator/wrapper_orchestrator.go

```diff
@@ -41,20 +38,16 @@ import (
 type WrapperOrchestrator struct {
 	// scaleUpRegularPods indicates that ScaleUp for regular pods will be run in the current CA loop, if they are present.
 	scaleUpRegularPods  bool
-	scaleUpOrchestrator scaleup.Orchestrator
+	podsOrchestrator    scaleup.Orchestrator
 	provReqOrchestrator scaleup.Orchestrator
 }
 
 // NewWrapperOrchestrator return WrapperOrchestrator
-func NewWrapperOrchestrator(kubeConfig *rest.Config) (scaleup.Orchestrator, error) {
-	provReqOrchestrator, err := checkcapacity.New(kubeConfig)
-	if err != nil {
-		return nil, fmt.Errorf("failed create ScaleUp orchestrator for ProvisioningRequests, error: %v", err)
-	}
+func NewWrapperOrchestrator(provReqOrchestrator scaleup.Orchestrator) *WrapperOrchestrator {
```

Comment:
WrapperOrchestrator consist of podsOrchestrator and provReqOrchestrator, since there is only one option for podsOrchestrator (at least for now), it doesn't make sense to have a parameter for podsOrchestrator. However provReqOrchestrator could be different depends on which provReqClasses are supported, so I added a parameter for it.

Reply:
Fair enough, this makes sense. Thanks for the explanation!